pontoon/base/views.py

```diff
@@ -123,7 +123,7 @@ def locale_permissions(request, locale):
     translators = l.translators_group.user_set.exclude(pk__in=managers).all()
     all_users = User.objects.exclude(pk__in=managers).exclude(pk__in=translators).exclude(email='')
 
-    contributors = User.translators.filter(translation__locale=l).distinct()
+    contributors = User.translators.filter(translation__locale=l).values_list('email', flat=True).distinct()
```

Comment:
Same as above: That was actually a bug. For project permissions, we compared a user dict to user objects, so we never had any matches.